hugo/content/blog/2025-12-27-finding-foss-bugs.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Bug Hunting in FOSS, with Mumble!
+author: abstractappsec(sam)
+date: 2025-12-27
+categories:
+  - News
+
+---
+
+In Mumble 1.5.857, there was a patch for an interesting bug:
+
+https://github.com/mumble-voip/mumble/pull/6875
+
+This patch was due to an experimental bug hunt that I began a few months ago, and for which I coordinated findings with @krzmbl. 
+In this blog post the developers invited me to present, I'll be taking a little detour from your standard Mumble blogs and talking about a way to contribute to open source even if you're not a particularly strong coder. 
+
+
+One of the fun things about cybersecurity is that a large part of it is thinking outside of the box.
+I work in application security and while I'm not super confident in my coding skills, I have noticed I do have a knack for finding bugs. Not necessarily those crazy bugs you hear about in the news, but I can find some strange ways to make software do things it's not normally expected to do.
+
+This idea and knack is how I ended up digging through the Mumble source code and discovering an interesting bug.
+
+I picked Mumble as my first software to analyze for the open source community because of two important factors. 
+
+- I have used Mumble on and off and I think it’s a great tool.
+- It isn’t one of those big fancy open source projects that is high vis, so I felt the developers might appreciate having some extra application security analysis for free.
+
+## Research
+When I decided to start doing some bug hunting, my first goal was to select a target “software under test.” I wanted something that fit into the following categories:
```

Comment:
I'm a bit confused. Isn't this section repeating what was said right before it? The reason why you chose Mumble?

I would drop the mention before this section, and then change the headline here from Research to *Picking Mumble for Analysis* or *Exploring Projects to Contribute to* or similar.